mocca-client/src/test/java/com/paypal/mocca/client/MoccaSerializerTest.java

```diff
@@ -141,6 +143,119 @@ public void complexResponseCustomSelectionSetLessFieldsTest() throws IOException
                 selectionSet, "{\n  \"query\" : \"query{getABeer(sampleRequest: {bar: \\\"bar\\\", foo: \\\"foo\\\"}) {booleanVar complexField stringVar}}\"\n}");
     }
 
+    @Test
+    public void complexWithStringListRequestTest() throws IOException {
+        SuperComplexSampleType.SuperComplexField superComplexField =
+                new SuperComplexSampleType.SuperComplexField(77, "sevenseven", false,
+                        Arrays.asList(new String[] {"cat", "dog", "monkey"}), null, null);
+        List<SuperComplexSampleType.SuperComplexField> listOfComplexSampleTypes = new ArrayList<>();
+        SuperComplexSampleType superComplexSampleType = new SuperComplexSampleType(7, "seven", true,
+                superComplexField, listOfComplexSampleTypes, null);
+
+        List<MoccaSerializer.Variable> variables = Collections.singletonList(
+                new MoccaSerializer.Variable( superComplexSampleType, SuperComplexSampleType.class, newVar("sampleRequest")));
+
+        requestTest(variables, SampleResponseDTO.class, "getOneComplexSample", MoccaUtils.OperationType.Mutation,
```

Comment:
This doesn't affect the test result, but just to be consistent, since you are using `getOneComplexSample`, this should be a `MoccaUtils.OperationType.Query`, not `MoccaUtils.OperationType.Mutation`.

Please do the same change in the other tests you added.